enterprise/admin/views.py

```diff
@@ -199,8 +200,19 @@ def _enroll_users(cls, emails, course_details, mode, request):
             ).format(enrolled_count=enrolled_count, course_id=course_id))
         if non_existing:
             messages.warning(request, _(
-                "The following users do not have accounts yet and were not enrolled: {}"
+                "The following users do not have accounts yet and were not yet enrolled: {}"
             ).format(", ".join(non_existing)))
+            for email in non_existing:
+                pending_user = PendingEnterpriseCustomerUser.objects.get(
```

Comment:
No; all pending ECUs should have already been created at this point. An email address in the non_existing list that hasn't been created as a pECU yet is a sign of an upstream logic failure.